PiicoDev_Unified.py

```diff
@@ -141,6 +161,16 @@ def write8(self, addr, reg, data):
     def read16(self, addr, reg):
         regInt = int.from_bytes(reg, 'big')
         return self.i2c.read_word_data(addr, regInt).to_bytes(2, byteorder='little', signed=False)
+    
+    def scan(self): #From Adafruit Blinka generic_linux
```

Comment:
Maybe move the comment into a docstring? It is a little messy inline imo but that's debatable

Reply:
Agreed - moving forward docstrings will be preferred for functions and methods. Anything (sustainable + non-destructive) we can implement to become more pythonic.
ToDo: verify they are stripped out by `minify` - though I expect they will be.